pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -7,13 +7,16 @@ import "../../../tasks/skylab/RunEmptyDrops.wdl" as RunEmptyDrops
 import "../../../tasks/skylab/CheckInputs.wdl" as OptimusInputChecks
 import "../../../tasks/skylab/MergeSortBam.wdl" as Merge
 import "../../../tasks/skylab/H5adUtils.wdl" as H5adUtils
+import "../../../tasks/broad/Utilities.wdl" as utils
```

Comment:
I am nervous about reusing this code here. Is it true we're only import the utils code to make a warning message? This is will now tie most of our skylab tests to changes that happen in the GP pipelines. I worry for Smart-tests.

Reply:
After talking about this in standup, we took a look and it looks like we do not touch the utilities wdl often, so we don't think this will be a big problem for our smart tests. If it becomes a problem, we can change it.